2_process.R

```diff
@@ -196,36 +268,43 @@ p2b_targets_list <- list(
   # the network due to odd PRMS-SNTemp values and so Barclay et al. (In Prep, 2023)
   # predicts temperature on 455 reaches in the DRB. 
   tar_target(
-    p2b_static_inputs_nhm_combined,
-    p2b_static_inputs_nhm_formatted %>%
-      left_join(y = p2a_static_inputs_prms, by = "seg_id_nat") %>%
-      left_join(y = p2b_soller_coarse_sediment_reaches_nhm  %>%
+    p2_static_inputs_nhm_combined,
+    p2_static_inputs_nhm_formatted_for_merge %>%
+      left_join(y = p2_static_inputs_prms, by = "seg_id_nat") %>%
+      left_join(y = p2_soller_coarse_sediment_reaches_nhm  %>%
```

Comment:
If you agree that we can safely omit four targets at the top of this file, we could edit this target to omit the reference to `p2_static_inputs_nhm_formatted_for_merge`:

```suggestion
    p2_static_inputs_nhm_combined,
    p2_static_inputs_prms %>%
      left_join(y = p2_soller_coarse_sediment_reaches_nhm  %>%
```